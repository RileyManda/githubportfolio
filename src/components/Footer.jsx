import { FaTwitter, FaGithub, FaStackOverflow, FaLinkedin, FaMedium } from 'react-icons/fa';
import Stack from 'react-bootstrap/Stack';

const Footer =()=> {
  return (
      <div>  <footer className="footer">
          <Stack direction="horizontal" gap={3}>
              <div className="p-2">
                  <a href="https://github.com/RileyManda/" target="_blank" rel="noopener noreferrer">
                      <FaGithub style={{ color: 'white' }} />
                  </a>
              </div>
              <div className="p-2">
                  <a href="https://stackoverflow.com/users/6129553/rileymanda" target="_blank" rel="noopener noreferrer">
                      <FaStackOverflow style={{ color: 'white' }} />
                  </a>
              </div>
              <div className="p-2">
                  <a href="https://www.linkedin.com/in/rileymanda/" target="_blank" rel="noopener noreferrer">
                      <FaLinkedin style={{ color: 'white' }} />
                  </a>
              </div>

              <div className="p-2">
                  <a href="https://twitter.com/rileycodez" target="_blank" rel="noopener noreferrer">
                      <FaTwitter style={{ color: 'white' }} />
                  </a>
              </div>
              <div className="p-2">
                  <a href="https://medium.com/@rileymanda" target="_blank" rel="noopener noreferrer">
                      <FaMedium style={{ color: 'white' }} />
                  </a>
              </div>


          </Stack>
          <p>Lets Connect and build some amazing projects</p>
      </footer>
      </div>
  )
}
export default Footer;
