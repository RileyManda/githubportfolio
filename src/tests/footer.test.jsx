import { describe,expect,test } from 'vitest';
import Footer from '../components/Footer';
import '@testing-library/jest-dom';
import { render } from '@testing-library/react';
// test suite
describe('Footer', () => {
    // footer should render correctly
  test('renders correctly', () => {
    const { container } = (<Footer />);
    expect(container).toMatchSnapshot();
  });
});
