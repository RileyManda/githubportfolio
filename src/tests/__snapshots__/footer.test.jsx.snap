// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Footer > renders correctly 1`] = `undefined`;
